deploy-java/action.yml

```
@@ -138,6 +138,19 @@ runs:
         jiraUsername: ${{ inputs.jiraUsername }}
         jiraPassword: ${{ inputs.jiraPassword }}
 
+    - name: On success notify okay to test
+      uses: actions/github-script@v6
+      if: ${{ (inputs.environment !='Production') }}
+      with:
+        github-token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+        script: |
+          const result = await github.rest.actions.createWorkflowDispatch({
+            owner: 'cartoncloud',
+            repo: 'api-functional-tests',
+            workflow_id: 'run-tests.yml',
+            environment: ${{ inputs.environment }}
```

Something like this may work
```suggestion
        script: |
          const environmentMappings = {
            'QA 01': 'qa1',
            'QA 02': 'qa2',
            'QA 03': 'qa3',
            'Test 01': 'test1',
            'Staging 01': 'staging1',
          };
          const result = await github.rest.actions.createWorkflowDispatch({
            owner: 'cartoncloud',
            repo: 'api-functional-tests',
            workflow_id: 'run-tests.yml',
            inputs: {
              environment: environmentMappings['${{ inputs.environment }}']
            }
```